05/hmw-05-01.py

```diff
@@ -0,0 +1,13 @@
+text = input('Input some text here: ')
+
+i = 0
+while i < len(text):
```

Comment:
The other approach you can use here is `for ... in`:
```
for c in text:
```
In this case, you will have every separate character of the text in variable `c` one by one each iteration